starter-code/src/app/contact-list/contact-list.component.ts

```diff
@@ -8,7 +8,7 @@ import contactList from '../contacts';
 })
 export class ContactListComponent implements OnInit {
   contacts: Object[];
-  newContact: Object = {};
+  newContact: Object = {name: ''};
```

Comment:
Maybe you could create an interface or class for a contact, to make it more strict and explicit?